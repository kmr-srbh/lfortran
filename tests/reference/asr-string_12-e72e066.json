{
    "basename": "asr-string_12-e72e066",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/string_12.f90",
    "infile_hash": "ba0e8d4d5c264402806c61ad398d62eaf5c9ddd47d86b173e5210342",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-string_12-e72e066.stdout",
    "stdout_hash": "a08e6307817e47782173c6d5441ecd424da6a7ef85c49869f1c16c40",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}