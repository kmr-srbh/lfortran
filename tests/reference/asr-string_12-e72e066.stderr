semantic error: The RHS character len must be known at compile time
 --> tests/../integration_tests/string_12.f90:4:5
  |
4 |     character(len=*), parameter :: fullhex_digits = "0123456789ABCDEFabcdef" !! 0 .. 9A .. Fa .. f
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
